src/twisted/web/test/test_http.py

```diff
@@ -3412,10 +3412,36 @@ def test_addCookieSameSite(self):
         self._checkCookie(
             b"foo=bar; SameSite=strict", b"foo", b"bar", sameSite="strict"
         )
+        self._checkCookie(
```

Comment:
I think that we need a separate test case with more details about what is tested here.

The test might be correct, but I find it hard to read

```suggestion

    def test_addCookieSameSiteSecure(self):
        """
        L{http.Request.setCookie} supports the C{sameSite} argument together with secure.
        It will set the cookie as ADD MORE DETAILS ABOUT THE TEST CASE.
        """
        self._checkCookie(
```